packages/application-shell/src/components/user-settings-menu/user-settings-menu.tsx

```diff
@@ -1,19 +1,22 @@
-import type { ControllerStateAndHelpers, DownshiftProps } from 'downshift';
-import type { TUser } from '../../types/generated/mc';
-import type { TFetchApplicationsMenuQuery } from '../../types/generated/proxy';
-
 import { ReactNode, useCallback, useEffect, useRef, useState } from 'react';
```

Comment:
Perhaps we can add the redesign comment to this file:
```
// TODO: @redesign cleanup
```

Reply:
Good point, thanks